test/unittests/eof_validation_test.cpp

```diff
@@ -301,19 +275,24 @@ TEST_F(eof_validation, EOF1_invalid_section_0_type)
 
 TEST_F(eof_validation, EOF1_too_many_code_sections)
 {
-    auto eof_code_sections_1024 = eof_bytecode(jumpf(1));
-    for (int i = 1; i < 1023; ++i)
-        eof_code_sections_1024 =
-            eof_code_sections_1024.code(jumpf(static_cast<uint16_t>(i + 1)), 0, 0x80, 0);
+    auto eof_code_sections_1023 = eof_bytecode(jumpf(1));
+    for (int i = 1; i < 1022; ++i)
+        eof_code_sections_1023 =
+            eof_code_sections_1023.code(jumpf(static_cast<uint16_t>(i + 1)), 0, 0x80, 0);
+
+    auto eof_code_sections_1024 = eof_code_sections_1023;
+    eof_code_sections_1023 = eof_code_sections_1023.code(OP_STOP, 0, 0x80, 0);
+    eof_code_sections_1024 = eof_code_sections_1024.code(jumpf(1023), 0, 0x80, 0);
```

Comment:
```suggestion
    const auto eof_code_sections_1024 = eof_code_sections_1023.code(jumpf(1023), 0, 0x80, 0);
```

Reply:
The `code` methos also modifies `eof_code_sections_1023`. This would add the `JUMPF(1023)` as the last section in eof_code_sections_1023, but the last section should contain `OP_STOP`.

Reply:
you can do this after
```suggestion
    const auto eof_code_sections_1024 = eof_code_sections_1023.code(jumpf(1023), 0, 0x80, 0);
    eof_code_sections_1023 = eof_code_sections_1023.code(OP_STOP, 0, 0x80, 0);
```

Reply:
Ah I see `eof_code_sections_1024` is also modified below, so `const` won't work.